ert3/engine/_run.py

```diff
@@ -0,0 +1,33 @@
+import ert3
+
+from pathlib import Path
+import random
+
+
+def _generate_coefficients():
+    return [
+        {
+            "coefficients": {
+                "a": random.gauss(0, 1),
+                "b": random.gauss(0, 1),
+                "c": random.gauss(0, 1),
+            }
+        }
+        for _ in range(1000)
+    ]
+
+
+def run(workspace_root, experiment_name):
+    experiment_root = Path(workspace_root) / experiment_name
```

Comment:
Looks like the variable experiment_root is no longer in use?

Reply:
You are right! I think it is a remnant from before using storage! Good catch 👍